rocky/requirements-dev.txt

```diff
@@ -255,6 +255,9 @@ django-admin-auto-tests @ git+https://github.com/dekkers/django-admin-auto-tests
 django-appconf==1.0.6 ; python_version >= "3.10" and python_version < "4.0" \
     --hash=sha256:c3ae442fba1ff7ec830412c5184b17169a7a1e71cf0864a4c3f93cf4c98a1993 \
     --hash=sha256:cfe87ea827c4ee04b9a70fab90b86d704cb02f2981f89da8423cb0fabf88efbf
+django-components==0.80 ; python_version >= "3.10" and python_version < "4.0" \
```

Comment:
I haven't looked into this PR in detail yet, but I noticed that there's a `0.81` version released recently